doc/source/reshaping.rst

```diff
@@ -518,6 +518,31 @@ the prefix separator. You can specify ``prefix`` and ``prefix_sep`` in 3 ways
     from_dict = pd.get_dummies(df, prefix={'B': 'from_B', 'A': 'from_A'})
     from_dict
 
+Sometimes it will be useful to only keep k-1 levels of a categorical
+variable to avoid collinearity when feed the result to statistical models.
+You can switch to this mode by turn on ``remove_first``.
+
+.. ipython:: python
+
+    s = pd.Series(list('abcaa'))
+
+    pd.get_dummies(s, remove_first=True)
+
+    df = pd.DataFrame({'key': list('bbacab'), 'data1': range(6)})
+
+    pd.get_dummies(df['key'], remove_first=True)
+
+You can also use ``remove_first`` as a equivalent to the ``omit.constants``
```

Comment:
Actually what I mean is that columns with only one level will be omitted in the results. An example I wrote in the tutorial 

```Python
In [91]: df = pd.DataFrame({'A':list('aaaaa'),'B':list('ababc')})

In [92]: pd.get_dummies(df, drop_first=True)
Out[92]: 
   B_b  B_c
0    0    0
1    1    0
2    0    0
3    1    0
4    0    1
```
I will make this clear. And let's discuss the behavior when we see a column with only one level.